go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/Netflix/go-expect v0.0.0-20220104043353-73e0943537d2
 	github.com/alicebob/miniredis/v2 v2.23.0
+	github.com/cenkalti/backoff/v4 v4.1.3
```

Comment:
The v3 version of this library was already a transitive dependency, but I figured we may as well use the latest major version.

This library does more than we need. I'm not a fan of the `Retry` functions it provides, but I think the `BackOff` interface and the implementations are nice. We could pretty easily inline some of the implementations or replace the library with our own implementations, but for now it seems like a good fit.